benchmarks/asv.conf.json

```diff
@@ -0,0 +1,80 @@
+{
+    // The version of the config file format.  Do not change, unless
+    // you know what you are doing.
+    "version": 1,
+
+    // The name of the project being benchmarked
+    "project": "cartopy",
+
+    // The project's homepage
+    "project_url": "https://github.com/scitools/cartopy",
+
+    // The URL or local path of the source code repository for the
+    // project being benchmarked
+    "repo": "../",
+
+    // Customizable commands for building, installing, and
+    // uninstalling the project. See asv.conf.json documentation.
+    //
+    // "install_command": ["python -mpip install {wheel_file}"],
+    // "uninstall_command": ["return-code=any python -mpip uninstall -y {project}"],
+    "build_command": [
+        "python setup.py build",
+        "PIP_NO_BUILD_ISOLATION=false python -mpip wheel --no-deps --no-index -w {build_cache_dir} {build_dir}"
```

Comment:
Don't understand why you need all these commands.

Reply:
Me either. I agree that it isn't needed. I just checked - the default is to ``python setup.py build`` followed by a ``python -mpip wheel`` then install with ``python -mpip install``.

I'll remove this block.